test/fixtures/plugin-invalid-https-url/manifest.json

```diff
@@ -11,5 +11,15 @@
       "http://example.com/http-is-invalid.js"
     ]
   },
+  "provider": {
+    "en": {
+      "name": "1",
+      "url": "https://jp.example.com"
+    },
+    "ja": {
+      "name": "",

```

Comment:
This file is for testing invalid HTTPS URL. So I think you should give the "name" property a valid value. E.g sample provider.